src/input.ts

```diff
@@ -92,49 +92,11 @@ export default class GlideCoreInput extends LitElement implements FormControl {
 
   static override styles = styles;
 
-  @property({ reflect: true })
```

Comment:
Alphabetized everything. We can be sure I didn't accidentally delete a property because the top-level JSDoc hasn't changed.